lib/good_job/log_subscriber.rb

```diff
@@ -243,13 +258,15 @@ def tag_logger(*tags, &block)
     #
     %w(info debug warn error fatal unknown).each do |level|
       class_eval <<-METHOD, __FILE__, __LINE__ + 1
-        def #{level}(progname = nil, tags: [], &block)   # def info(progname = nil, tags: [], &block)
-          return unless logger                           #   return unless logger
-                                                         #
-          tag_logger(*tags) do                           #   tag_logger(*tags) do
-            logger.#{level}(progname, &block)            #     logger.info(progname, &block)
-          end                                            #   end
-        end                                              #
+        def #{level}(progname = nil, tags: [], &block)          # def info(progname = nil, tags: [], &block)
+          return unless logger                                  #   return unless logger
+                                                                #
+          tag_logger(*tags) do                                  #   tag_logger(*tags) do
+            logger.#{level}(progname, &block)                   #     logger.info(progname, &block)
+          end                                                   #   end
+                                                                #
+          $stdout.flush if logging_to_stdout? && !$stdout.sync  #   $stdout.flush if logging_to_stdout? && !$stdout.sync
```

Comment:
Not sure if this is the best place to flush the $stdout buffer. Let me know what you think.